README.md

```diff
@@ -61,3 +61,22 @@ curl -X POST -d "grant_type=password&username=<user_name>&password=<password>&sc
 ```
 
 Now, this token will have read-only access to the API.
+
+
+## Testing
+
+1. Install [docker](http://docs.docker.com/installation/ubuntulinux/) and
+   [docker-compose](https://docs.docker.com/compose/install/)
+2. From the root project directory, run `docker-compose run test` (or `docker-compose
+   up`). You may need to use sudo depending on how you've configured Docker.
+
+If you encounter connection-refused errors, try re-running the test command;
+docker-compose will launch the postgres container first, but sometimes postgres doesn't
+become available quickly enough for Django to connect to it. Re-running will usually work.
```

Comment:
Unfortunately not in docker-compose itself: https://github.com/docker/compose/issues/374

I'll see if there's an elegant way to make run_tests sleep or retry a failed connection. The only time this error should occur is if you destroy the db container without destroying the test container. Then the test container will boot really fast but the new db container will take a couple seconds to boot the first time.